docs/sql-ref-syntax-qry-window.md

```diff
@@ -19,4 +19,222 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+Similarly to aggregate functions, window functions operate on a group of rows. However, unlike aggregate functions, window functions perform aggregation without reducing, calculating a return value for each row in the group. Window functions are useful for processing tasks such as calculating a moving average, computing a cumulative, or accessing the value of rows given the relative position of the current row. Spark SQL supports three types of window functions:
+  * Ranking Functions
+  * Analytic Functions
+  * Aggregate Functions
+
```

Comment:
How about putting the `Syntax` section here before the `How to Use Window Functions` section?

Reply:
The ```How to Use Window Functions``` is removed.